### ISSUE TITLE
Git pull/checkout/clone fails on Windows

### ISSUE THREAD

**Issue by ethindp** [EOSIO/eos-vm#139]
Attempting to integrate this into CMake with the ExternalProject module. Git pull fails with various errors on Windows only; files (pretty much tests) include characters that give "invalid argument" errors (mainly colons and other characters disallowed by Windows in filenames). Is there a way this can either be fixed or ignored? As a result of this, git fails with the following error when pulling:
```
fatal: Could not reset index file to revision 'HEAD'.
Cannot rebase: You have unstaged changes.
Please commit or stash them.
No rebase in progress?
```

**Comment by ethindp** [comment]
Update: I could do a sparse checkout, but I don't want to affect the tests. If I ignore the tests folder, will CMake still be able to build the project?

**Comment by larryk85** [comment]
Yes, you can pass `-DENABLE_TESTS=Off` to the command line and this will ignore the tests.  But, currently we only support Unixish machines, so Windows would need work to handle memory allocations and the way we do guard paging and timer failures.
